code/iaas/service-discovery/server/src/main/java/io/cattle/platform/servicediscovery/service/impl/RancherConfigToComposeFormatter.java

```diff
@@ -0,0 +1,7 @@
+package io.cattle.platform.servicediscovery.service.impl;
+
+import io.cattle.platform.servicediscovery.resource.ServiceDiscoveryConfigItem;
+
+public interface RancherConfigToComposeFormatter {
+    public Object format(ServiceDiscoveryConfigItem item, Object valueToTransform);
```

Comment:
Ah.  I'm good with that too.  I guess if we can document the interface, that'd be great!